pages/linux/qm-destroy.md

```diff
@@ -0,0 +1,20 @@
+# qm destroy
+
+> Destroy a virtual machine (VM) in QEMU/KVM Virtual Machine Manager.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Destroy a specific VM:
+
+`qm destroy {{vm_id}}`
+
+- Destroy all disks with the same VMID that are not explicitly referenced in the configuration:
+
+`qm destroy {{vm_id}} --destroy-unreferenced-disks`
+
+- Remove VMID from all configurations such as backup & replication jobs:
```

Comment:
What means `remove from configurations`?

Reply:
Hi,
Agree that is confusing. That's how Proxmox is defining it on their [documentation](https://pve.proxmox.com/pve-docs/qm.1.html).
On their Support Forum a Proxmox Support Staff defined it by "it will also remove the VMID from backup jobs and so forth" ([link](https://forum.proxmox.com/threads/difference-between-destroy-purge-vm.100996/)).

Can I suggest: "Remove VMID from all locations (that includes inventory, backup jobs, HA...)"